pkg/store/volume.go

```diff
@@ -0,0 +1,69 @@
+package store
+
+import (
+	"errors"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/lima-vm/lima/pkg/qemu/imgutil"
+	"github.com/lima-vm/lima/pkg/store/filenames"
+)
+
+type Volume struct {
+	Name        string `json:"name"`
+	Size        int64  `json:"size"`
+	Dir         string `json:"dir"`
+	DiskPath    string `json:"diskPath"`
+	Instance    string `json:"instance"`
+	InstanceDir string `json:"instanceDir"`
+}
+
+func InspectVolume(volName string) (*Volume, error) {
```

Comment:
I wonder if it would make sense to follow the same API as the `store.Inspect()` function: to only return an error if the volume doesn't exists, and otherwise collecting errors in a `volume.Errors` slice.

It isn't really needed here, but it might be nice for consistency.